apps/website/get-started/angular.md

```diff
@@ -38,14 +38,14 @@ Clarity is published as three separate packages on NPM, as well as one dependenc
 - **@clr/ui.** Contains the static styles for building HTML components.
 - **@clr/core.** Contains a shared core library of web components.
 - **@clr/angular.** Contains the Angular components. This package depends on @clr/ui for styles.
-- **@webcomponents/webcomponentsjs.** A polyfill for webcomponents for older browsers, which Clarity depends upon.
+- **@webcomponents/webcomponentsjs.** A polyfill for web components for older browsers, which Clarity depends upon.
 
 Install them all by running the following command with `npm` or `yarn`.
 
 <DocDemo toggle="false">
 
 ```bash
-npm install @clr/icons @clr/angular @clr/ui @clr/core @webcomponents/webcomponentsjs --save
+npm install @clr/icons @clr/angular @clr/ui @clr/core @webcomponents/webcomponentsjs
```

Comment:
Why remove the save flag here?

Reply:
I've removed it because as of npm 5.0.0, installed modules are added as a dependency by default. I must have overwritten the commit message that clarified why I made the change.